Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	2	clean_indelible
	6	concat_fasta
	2	concat_indelible
	3	concat_summaries
	1	create_control_files
	1	create_plots
	6	run_grasp
	2	run_indelible
	6	summarise_indels
	29
Select jobs to execute...

[Sun Jan 31 18:16:31 2021]
rule create_control_files:
    output: indelible/6_control.txt
    jobid: 6
    wildcards: taxon=6

[Sun Jan 31 18:16:31 2021]
Finished job 6.
1 of 29 steps (3%) done
Select jobs to execute...

[Sun Jan 31 18:16:31 2021]
rule run_indelible:
    input: indelible/6_control.txt
    output: indelible_output/6/trees/1.nwk, indelible_output/6/6_TRUE_1.fasta, indelible_output/6/6_ANCESTRAL_1.fasta, indelible_output/6/N0_trees/1.nwk
    jobid: 5
    wildcards: taxon=6, rep=1

ImproperOutputException in line 15 of /Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/Snakefile:
Outputs of incorrect type (directories when expecting files or vice versa). Output directories must be flagged with directory(). for rule run_indelible:
indelible_output/6/N0_trees/1.nwk
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 583, in handle_job_success
  File "/Users/gabefoley/miniconda3_latest/envs/indel_snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 252, in handle_job_success
Removing output files of failed job run_indelible since they might be corrupted:
indelible_output/6/trees/1.nwk, indelible_output/6/6_TRUE_1.fasta, indelible_output/6/6_ANCESTRAL_1.fasta, indelible_output/6/N0_trees/1.nwk
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /Users/gabefoley/Dropbox/PhD/20210121_Indel_Evaluation_Project/Indel_Evaluation_Snakemake/.snakemake/log/2021-01-31T181630.970463.snakemake.log
